demos/array-shell-sort/TeX/bibliography.bib

```diff
@@ -0,0 +1,8 @@
+@inproceedings{goodrich2010randomized,
```

Comment:
I do not mean to be pedantic, but I remember there was a way to put the bib as part of the tex file, for a small bib like this one, I would prefer not to have a separate file and clutter the repo.

If you do not know how to the bib thing of the top of your head, then we will keep it this way, not very important, do not waste your time on it.

Reply:
Resolved with last push. Way to do this that I know does re-generate the bib file on compiling, but at least it doesn't need to be in the git repo.